skills/docx/ooxml/scripts/validation/base.py

```
@@ -0,0 +1,951 @@
+        errors = []
+
+        for xml_file in self.xml_files:
+            try:
+                # Try to parse the XML file
+                lxml.etree.parse(str(xml_file))
+            except lxml.etree.XMLSyntaxError as e:
+                errors.append(
+                    f"  {xml_file.relative_to(self.unpacked_dir)}: "
+                    f"Line {e.lineno}: {e.msg}"
+                )
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)